src/components/Content.js

```diff
@@ -0,0 +1,24 @@
+import { Home, About, Services, Team, Appointment, FAQ, Contact, PersDataProt, PrivacyPolicy } from "../pages/index";
+
+import { Switch, Route } from "react-router-dom";
+import { routes } from "../routing/routes";
+
+const Content = () => {
```

Comment:
Name Content is confusing for me, I would expect to have here some JSX which is rendered as main part of page/app. Maybe better would be AppRouting/AppRouter ? Name that clearly say that this file contains routing.

Reply:
Changed name to AppRouting.